### ISSUE TITLE
Syntax Highlighting Issue after v1.2+

### ISSUE THREAD

**Issue by superguineapig** [walles/moar#86]
### Description
First, I love `moar`.  I've been using it for a couple of years now as my primary pager, and it's great!

I recently wound up re-installing `moar` through `homebrew`, without errors or warnings, and now all languages colorize the same way; with strange blue/gray highlighting:

Here's a `.go` file:
<img width="962" alt="example_go" src="https://user-images.githubusercontent.com/4808737/160920122-360a2bc1-61c0-4c89-90cd-59cae3b34e89.png">
a `.json` file
<img width="962" alt="example_json" src="https://user-images.githubusercontent.com/4808737/160920157-40dccce9-f5e6-4f98-a1aa-a39b0e657298.png">
a `.js` file
<img width="962" alt="example_js" src="https://user-images.githubusercontent.com/4808737/160923939-89d3c0a6-cda9-43ef-94ff-5256c7def706.png">
a `.cc` file
<img width="962" alt="example_cc" src="https://user-images.githubusercontent.com/4808737/160924011-626c72e0-3a7e-4f08-bb86-4434e6bbdfc7.png">

### Additional testing
I also tried building direct from source for all versions below latest until it worked again.
Lo and behold, `tag 1.2` produced the correct syntax highlighting. Apparently, my previous version must have been at or before 1.2.

a `.go` file viewed with `moar` @ 1.2 (same terminal and env)
<img width="962" alt="example_go_1_2" src="https://user-images.githubusercontent.com/4808737/160928921-1e04ff96-2616-4bea-b59f-6788642d60c4.png">

Really not sure what my setup doesn't like about versions >1.2

Any guidance would be appreciated.

### Details
Mac OS: 12.3
Mac Terminal.app: 2.12.5 (default)
Moar: 1.9.0 as installed via Homebrew, or built from source; all tags >1.2 
Homebrew: 3.4.4
Go: 1.17.8
Shell: bash
ENV:
```
TERM=xterm-256color
LANG=en_US.UTF-8
MOAR=""
```
```
$ locale
LANG="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_CTYPE="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_ALL=
```

Thank you!

**Comment by walles** [comment]
Thank you for your excellent analysis @superguineapig ⭐.

I get the exact same symptoms using the built-in Terminal application, haven't figured out why yet, but I confirm this.

Personally I'm on iTerm, which is why I never noticed.

**Comment by walles** [comment]
Running with `moar --colors=256 moar.sh` works around this problem.

This is a workaround rather than a solution, but even so it is a step in the right direction.

**Comment by walles** [comment]
Fixed in `v1.9.1`, should be out in Homebrew soon: https://github.com/Homebrew/homebrew-core/pull/98352

**Comment by superguineapig** [comment]
@walles looks good.  Thanks for looking at this so quickly!
